apps/docs/content/guides/auth/social-login/auth-google.mdx

```diff
@@ -1,26 +1,19 @@
 ---
 id: 'auth-google'
 title: 'Login with Google'
-description: 'Use Sign in with Google on the web, in native apps or with Chrome extensions'
+description: 'Use Sign In with Google on the web, in native apps or with Chrome extensions'
```

Comment:
Is this the new styling we apply? Maybe we should leave it for another PR where all `Sign in` is renamed to `Sign In`?

Your call!

Reply:
Yeah I'll revert this 👍🏼